mica/stats/acq_stats.py

```diff
@@ -475,12 +469,15 @@ def warn_on_acq_anom(acqs, emails):
             text = "Does not appear to be classic star-in-wrong-box anomaly\n"
         # Make a dictionary of the anom record for use in string formatting
         output_dict = {col: anom[col] for col in anom.colnames}
-        output_dict['dy'] = anom['yang_obs'] - anom['yang']
-        output_dict['dz'] = anom['zang_obs'] - anom['zang']
+        output_dict['dy'] = anom['dy']
+        output_dict['dz'] = anom['dz']
         text += """Large Deviation from Expected ACQ Star Position in {obsid}
-      Slot {slot} Expected (Y-Pos, Z-Pos) = ({yang:.1f}, {zang:.1f})
+      Slot {slot} Expected* (Y-Pos, Z-Pos) = ({yang:.1f}, {zang:.1f})
```

Comment:
Is that `*` a "fat-finger"?  (Brent's word).

Reply:
I really was trying for a "see the asterisk note below", so it isn't a "fat-finger", but it looks like it wasn't effective.

Reply:
Yeah, just skip the asterisk, it is confusing.